src/esperanto/NewsletterBundle/Resources/config/doctrine/Newsletter.orm.yml

```diff
@@ -0,0 +1,10 @@
+esperanto\NewsletterBundle\Entity\Newsletter:
+    type: entity
+    table: null
```

Comment:
table name is newsletter_newsletter. First the application name and then the resource.